api/src/main/java/se/fortnox/reactivewizard/CollectionOptions.java

```diff
@@ -20,7 +20,7 @@
     @QueryParam("order")
     private SortOrder order;
 
-    private boolean lastRecord = true;
+    private boolean lastRecord = false;
```

Comment:
Is it possible to test the desired behaviour? A test with a streaming result and two pages should do?